gui/zoomableview.h

```diff
@@ -58,6 +59,13 @@ public slots:
     /// Provides support for zoom-in/out with a mouse while holding Control key.
     void wheelEvent(QWheelEvent *event) override;
 
+    /// Provides support for starting the panning of the window using the left
+    /// mouse button
+    void mousePressEvent(QMouseEvent *event) override;
```

Comment:
Ideally, these overrides should belong to the DiagramView instead of the ZoomableView,
but it is good enough as is.
I'll have to separate the DiagramView into its own module in future.